frontend/src/views/supp-allowances/SupplementaryFormView.vue

```diff
@@ -105,12 +168,51 @@ export default {
       },
     ]
     this.panel = this.allPageIDs
+    await this.loadData()
+    this.loading = false
   },
   methods: {
+    ...mapActions(useApplicationsStore, ['getSupplementaryApplications']),
     isEmpty,
     togglePanel() {
       this.panel = isEmpty(this.panel) ? this.allPageIDs : []
     },
+    async loadData() {
+      try {
+        this.setUpDefaultNewRequestModel(await this.getSupplementaryApplications(this.applicationId))
+      } catch (error) {
+        this.setFailureAlert('Failed to load supplementary applications')
+      }
+    },
+    setUpDefaultNewRequestModel(suppApplications) {
+      const indigenousProgrammingModel = {
+        indigenousFundingModel: [],
+        indigenousOtherDescription: null,
+        supplementaryApplicationId: undefined,
+        supplementaryType: 2,
```

Comment:
can we have a constant for this supplementary type? I think it would be easier to understand the code.